pkg/config/config.go

```diff
@@ -151,6 +151,7 @@ type GasEstimator interface {
 	Mode() string
 	PriceMaxKey(gethcommon.Address) *assets.Wei
 	EstimateLimit() bool
+	SenderAddress() *string
```

Comment:
Let's return an address instead of a string. Then we get validation once up-front during initialization and the callers don't have to worry about it.
```suggestion
	SenderAddress() *gethcommon.Address
```